proposals/solid-oidc-primer/index.bs

```diff
@@ -413,7 +417,8 @@ Token Header:
 }
 ```
 
-- `"alg": "ES256"`: The signing algorithm used in this token. In this case, `ES256` because we generated the keys using eliptic curves.
+- `"alg": "ES256"`: The signing algorithm used in this token. In this case, `ES256` 
+  because we generated the keys using eliptic curves.
```

Comment:
```suggestion
- `"alg": "ES256"`: The signing algorithm used in this token. In this case, `ES256` 
   because we generated the keys using eliptic curves.
```